src/main/webapp/app/shared/sort/sort-icon.component.ts

```diff
@@ -0,0 +1,21 @@
+import { Component, computed, input } from '@angular/core';
+import { FontAwesomeModule } from '@fortawesome/angular-fontawesome';
+import { CommonModule } from '@angular/common';
+import { faSortDown, faSortUp } from '@fortawesome/free-solid-svg-icons';
+
+@Component({
+    selector: 'jhi-sort-icon',
+    templateUrl: './sort-icon.component.html',
+    styleUrls: ['./sort-icon.component.scss'],
+    standalone: true,
+    imports: [FontAwesomeModule, CommonModule],
+})
+export class SortIconComponent {
+    direction = input.required<'asc' | 'desc' | 'none'>();
```

Comment:
for 'asc' 'desc' etc. you could laso use an enum as you use it in several places already